extras/vfd16lf01/LICENSE

```diff
@@ -0,0 +1,7 @@
+This is a driver for Samsung 16LF01 series VFD.
+
+Originate written by Daniele Napolitano, distributed under GPL.
```

Comment:
Sorry, this is a problem..  We can't accept any code that's under the GPL, because it contaminates our own BSD licensing.  You'll need to find or make something under a more compatible license..